circle.yml

```diff
@@ -15,26 +16,28 @@ test:
     - RAILS_ENV=test bundle exec rspec -r rspec_junit_formatter --format RspecJunitFormatter -o $CIRCLE_TEST_REPORTS/rspec/junit.xml
 
 deployment:
+  cloud-migration:
   staging:
     branch: develop
     owner: AusDTO
     commands:
-      - cf api https://api.system.staging.digital.gov.au
-      - cf auth $CF_STAGING_USER $CF_STAGING_PASSWORD
-      - cf target -o govau -s platform
+      - cf api https://api.system.y.cld.gov.au
+      - cf auth $CF_USER_STAGING $CF_PASSWORD_STAGING
+      - cf target -o dta -s govau
       - curl https://api.rollbar.com/api/1/deploy/ -F access_token=$ROLLBAR_ACCESS_TOKEN
           -F environment=production -F revision=`git log -n 1 --pretty=format:"%H"`
           -F local_username=`whoami`
-      - ./bin/deploy-staging.sh
+      - bundle exec rake assets:precompile
```

Comment:
@maxious @gordcorp are there any down-sides to running `rake assets:precompile` in CI?  
it's typically done by the CF ruby build pack while staging the application, but devise complains about a missing env-var that doesn't existing in the staging.  Running it hear means CF doesn't try to.

Reply:
There are some catches about relying on services but I don't think they should matter to this app http://docs.cloudfoundry.org/buildpacks/ruby/ruby-tips.html#precompile